.gitignore

```diff
@@ -10,3 +10,6 @@ config.js
 # IntelliJ specific project files
 .idea/
 portfolio.iml
+
+# vim swap files
+**/*.swp
```

Comment:
I prefer small, independent fixes like this to go into separate PRs, which can even skip review from me/Julian. It's fine to do occasionally for convenience, but prefer to send them on their own in general. 

They're easier to review quickly, and don't "litter" the actual point of the PR. See:
https://drive.google.com/file/d/1CNUwfkvB00ZPeAb7aVssF9mIYINzumeZ/view?usp=sharing

You can just reply ACK and resolve this comment.

Reply:
ACK